nilearn/plotting/displays.py

```diff
@@ -95,7 +98,7 @@ def draw_2d(self, data_2d, data_bounds, bounding_box,
                                **kwargs)
 
         self.add_object_bounds((xmin_, xmax_, zmin_, zmax_))
-
+      
```

Comment:
I know it's not intentional ;). You should take a look at vim-flake8 that spot these problems.